CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.3.7] - 2022-07-25
+
+- chore:
+  - update Node to `v18`
+  - update node-utils to `v1.0.0`
+- fix:
+  - resolve eslint errors
+  - reorder core-dump (alphabetically institutions)
+- feat:
+  - update docker version tag from `package.json`
+  - add eslint check to github-actions
```

Comment:
Noticed that we technically don't stick to our own recommended [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) schema.

Reply:
Possibly it might also be helpful to write all changes as 

```md
- chore: 123
- chore: 456
- fix: 789
- fix: 012
```

to allow automatic release note creation? 
https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes
https://github.com/bcoe/conventional-release-labels
https://www.conventionalcommits.org/en/v1.0.0/#specification

Reply:
Thats true, we should stick more to Conventional Commits then!